packages/ui/src/blocks/faqs/faq-data.tsx

```diff
@@ -59,10 +61,10 @@ export const FAQ_LIST: FAQItem[] = [
     icon: <IconCreditCard />,
   },
   {
-    title: `Can Chirpy be self-hosted?`,
+    title: `Is self-hosted supported for Chirpy?`,
     description: (
       <span>
-        {`Yes, it is. It's exactly the same product as our Cloud solution, but you have to install, host and manage your own infrastructure while the Cloud version we manage everything for your ease and convenience. `}
```

Comment:
The previous one is better, we don't want our customers prefer self-hosted version rather than our cloud version.